saleor/static/dashboard-next/shipping/components/ShippingZoneCreatePage/ShippingZoneCreatePage.tsx

```diff
@@ -0,0 +1,139 @@
+import * as React from "react";
+
+import CardSpacer from "../../../components/CardSpacer";
+import { ConfirmButtonTransitionState } from "../../../components/ConfirmButton";
+import Container from "../../../components/Container";
+import CountryList from "../../../components/CountryList";
+import Form from "../../../components/Form";
+import Grid from "../../../components/Grid";
+import PageHeader from "../../../components/PageHeader";
+import SaveButtonBar from "../../../components/SaveButtonBar";
+import Toggle from "../../../components/Toggle";
+import i18n from "../../../i18n";
+import { CountryFragment } from "../../../taxes/types/CountryFragment";
+import { UserError } from "../../../types";
+import ShippingZoneCountriesAssignDialog from "../ShippingZoneCountriesAssignDialog";
+import ShippingZoneInfo from "../ShippingZoneInfo";
+
+export interface FormData {
+  countries: string[];
+  default: boolean;
+  name: string;
+}
+
+export interface ShippingZoneCreatePageProps {
+  countries: CountryFragment[];
+  disabled: boolean;
+  errors: UserError[];
+  saveButtonBarState: ConfirmButtonTransitionState;
+  onBack: () => void;
+  onSubmit: (data: FormData) => void;
+}
+
+const ShippingZoneCreatePage: React.StatelessComponent<
+  ShippingZoneCreatePageProps
+> = ({ countries, disabled, errors, onBack, onSubmit, saveButtonBarState }) => {
+  const initialForm: FormData = {
+    countries: [],
+    default: false,
+    name: ""
+  };
+  return (
+    <Toggle>
+      {(
+        isCountrySelectionDialogOpened,
+        { toggle: toggleCountrySelectionDialog }
+      ) => (
+        <Form errors={errors} initial={initialForm} onSubmit={onSubmit}>
+          {({ change, data, errors: formErrors, hasChanged, submit }) => (
+            <>
+              <Container width="md">
+                <PageHeader
+                  title={i18n.t("Create New Shipping Zone")}
+                  onBack={onBack}
+                />
+                <Grid>
+                  <div>
+                    <ShippingZoneInfo
+                      data={data}
+                      errors={formErrors}
+                      onChange={change}
+                    />
+                    <CardSpacer />
+                    <CountryList
+                      countries={data.countries.map(selectedCountry =>
+                        countries.find(
+                          country => country.code === selectedCountry
+                        )
+                      )}
+                      disabled={disabled}
+                      emptyText={
+                        data.default
+                          ? i18n.t(
+                              "This is default shipping zone, which means that it covers all of the countries which are not assigned to other shipping zones"
+                            )
+                          : i18n.t(
+                              "Currently, there are no countries assigned to this shipping zone"
+                            )
```

Comment:
```
i18n.t(
  data.default
    ? "This is default shipping zone, which means that it covers all of the countries which are not assigned to other shipping zones"
    : "Currently, there are no countries assigned to this shipping zone"
);
```

Reply:
AFAIK not gonna work with https://github.com/i18next/i18next-scanner